### ISSUE TITLE
vSphere upgrade-opsmgr fails for while downloading stemcells 

### ISSUE THREAD

**Issue by nebhale** [pivotal-cf/pcf-pipelines#93]
Last night while trying to upgrade to Ops Manager 1.10.8, my pipelines (0.14.1, da072c72d12bd59d09f2561a8d9aafb276cf1cf5) failed in the `download-stemcells` task with the following error:

```
Logged-in successfully
pcf-pipelines/tasks/download-pivnet-stemcells/task.sh: line 55: pivnet: unbound variable
pcf-pipelines/tasks/download-pivnet-stemcells/task.sh: line 60: pivnet: unbound variable
Could not find stemcell 3363.24 for vsphere. Did you specify a supported IaaS type for this stemcell version?
```

**Comment by xyloman** [comment]
After upgrading my pipeline to 0.14.1 today I have seen the same issue however, with IaaS type AWS.
```Logged-in successfully
pcf-pipelines/tasks/download-pivnet-stemcells/task.sh: line 55: pivnet: unbound variable
pcf-pipelines/tasks/download-pivnet-stemcells/task.sh: line 60: pivnet: unbound variable
Could not find stemcell 3363.24 for aws. Did you specify a supported IaaS type for this stemcell version?```

**Comment by krishicks** [comment]
This will be fixed in the next release, 0.15.1+
